package.json

```diff
@@ -1,31 +1,31 @@
 {
-  "name": "public-website",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "ajv": "^6.10.0",
-    "bootstrap": "^4.3.1",
-    "node-sass": "^4.13.1",
-    "react": "next",
-    "react-bootstrap": "^1.0.0-beta.9",
-    "react-dom": "next",
-    "react-google-docs-viewer": "^1.0.1",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "^2.0.5"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
-}
+    "name": "public-website",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "ajv": "^6.10.0",
+        "bootstrap": "^4.3.1",
+        "node-sass": "^4.13.1",
+        "react": "next",
+        "react-bootstrap": "^1.0.0-beta.9",
+        "react-dom": "next",
+        "react-router-dom": "^4.3.1",
+        "react-scripts": "^2.0.5"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject",
+        "lint": "eslint \"**/*.js\" \"**/*.jsx\" --fix"
+    },
+    "eslintConfig": {
+        "extends": "react-app"
+    },
+    "browserslist": [
+        ">0.2%",
+        "not dead",
+        "not ie <= 11",
+        "not op_mini all"
+    ]
```

Comment:
I recommend using prettier-eslint since it's more powerful than just eslint alone (i.e. it can auto format fix code more strictly)

see usage here https://github.com/yayimahuman/Harry/blob/master/package.json#L27

also would need to `npm i --save-dev prettier-eslint prettier-eslint-cli`